libs/payments/ui/src/lib/client/components/CheckoutForm/index.tsx

```diff
@@ -292,6 +292,7 @@ export function CheckoutForm({
                     },
                     data.orderID
                   );
+                  router.push('./processing');
```

Comment:
Currently the action already performs the redirect. could you remove one of the redirects?

Reply:
Yes, I'll remove the redirect that the action performs. The page was staying on `start` for too long previously, causing a minor UX issue.

Reply:
removed this redirect. I was not able to get the UX issue solved around the redirect timing, so that may be something we want to take a closer look at. It doesn't seem like a call that should require 3 database round-trips before we actually call the  `payWith<Stripe|Paypal>` methods, but its an optimization that feels out of scope for this